cms/djangoapps/contentstore/views/videos.py

```diff
@@ -147,6 +156,25 @@ def videos_handler(request, course_key_string, edx_video_id=None):
         return videos_post(course, request)
 
 
+@expect_json
+@login_required
+@require_POST
+def video_images_handler(request, course_key_string, edx_video_id=None):
+    # TODO : We need to be sure what data jQuery fileuploader sends.
+    image_file = request.POST['file']
+    file_name = request.POST['file_name']
+
+    # TODO: Image file validation
```

Comment:
We will need to do image validation here. If any validation fails we would send an error response